examples/lusid/orders/order-blocks/create-custom-property-on-order-block/3-create-custom-property-in-block-domain.sql

```diff
@@ -0,0 +1,38 @@
+-- set property variables
+
+@@scope = SELECT 'blockUpdateExample';
+
+@@propertyCode = SELECT 'Contingent_Id';
+
+@@propertyDisplayName = SELECT 'Contingent Id';
+
+@@propertyDescription = SELECT 'A property representing the contingent ID of the Block';
+
+
+-- Create Property definition
+
+@table_of_data = SELECT 'Block' as Domain, 
+@@scope as PropertyScope, 
+'system' as DataTypeScope, 
+'string' as DataTypeCode, 
+'property' as ConstraintStyle, 
+@@propertyCode as PropertyCode, 
+@@propertyDisplayName as DisplayName, 
+'insert' as WriteAction;
+
+select * from Lusid.Property.Definition.Writer where ToWrite = @table_of_data;
+
+
+-- Apply Newly created property
```

Comment:
Comment should be "inline property" with a link to Luminesce

Reply:
added link to support page https://support.lusid.com/knowledgebase/article/KA-01702/